modules/io/common/src/main/java/org/locationtech/jts/io/geojson/TransformationUtils.java

```diff
@@ -0,0 +1,85 @@
+package org.locationtech.jts.io.geojson;
+
+import org.locationtech.jts.algorithm.Orientation;
+import org.locationtech.jts.geom.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TransformationUtils {
```

Comment:
How about calling this `OrientationTransformer`?

Reply:
I followed this recommendation 👍